pace-neutrons/Herbert
# dlpl issue

apparently its a feature, that Matlab plots only continuous lines if data are interrupted by NaN-s. 
If such situation happens, let's issue a warning informing user that he is not seeing number of points. 

To test:
go to  babylon5\Public\abuts\plot_bug\ and run script which loads data with Nans and demonstrate that line is not plotted. 
Switch to the branch and see warning message, explaining why data are not plotted.
